### PR TITLE
gradle clean up

### PR DESCRIPTION
**Pull request by StefMa**
This is just some gradle "clean up"

* Remove the buildscript{} block completely and replace with the newer plugins{} block
* Remove the google() repo and dependencies because we don't use it
* Update the bintray plugin to the latest version 0.8.0
* Cleared the top-level `build.gradle` because we don't need it for now...

I've tested it by pushing the version to mavenLocal() and used this version to publish a new version of ThirtyInch. Works without any issues. See https://dl.bintray.com/stefma/maven/net/grandcentrix/thirtyinch/ and search for the version `0.8.6-gradlecleanup/`.
I've also used that version in a current project without any issues 👍

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -33,11 +21,24 @@ compileGroovy {
...
+        }
+    }
+}
+
 publish {
     userOrg = 'novoda'
     groupId = 'com.novoda'
     artifactId = rootProject.name
-    version = project.version
+    version = "0.8.0"
```
Comment (passsy) [inline core/build.gradle]: This is now a duplicate to https://github.com/novoda/bintray-release/blob/master/build.gradle#L8

**Comment by StefMa** [comment]
The top level build.gradle is complete empty in this PR because we don't
need the settings in their. We have overwritten the repositories in the
core module anyway.

I've also removed the version from  their and hard-coded because we had two
different types of implementations. Using the version from the project but
hard codes the group name.

Instead we could also set the group name to the project (and the version of
course) and use it in the publish block.
But since we use it only once it make no sense to declare it for the
project....

File: core/build.gradle (hunk 1 above)
Comment (tasomaniac) [inline core/build.gradle]: Maybe have single quotes? Would be consistent.

**Comment by StefMa** [comment]
Fixed @tasomaniac 👍

**Pull request merged**